lua/go/codelens.lua

```diff
@@ -53,7 +53,7 @@ function M.refresh()
   if _GO_NVIM_CFG.lsp_codelens == true then
     local found = false
     if not found then
-      for _, lsp in pairs(vim.lsp.get_active_clients()) do
+      for _, lsp in pairs(vim.lsp.buf_get_clients(0)) do
```

Comment:
buf_get_clients is deprecated since nvim 0.9.x
You can add a filter to get_active_clients

```

    Parameters:  
      • {filter}  (table|nil) A table with key-value pairs used to filter the
                  returned clients. The available keys are:
                  • id (number): Only return clients with the given id
                  • bufnr (number): Only return clients attached to this
                    buffer
                  • name (string): Only return clients with the given name
```